pkgs/development/compilers/openjdk/jre.nix

```
@@ -7,7 +7,7 @@
 
 let
   jre = stdenv.mkDerivation {
-    name = "${jdk.name}-minimal-jre";
+    pname = "${jdk.pname}-minimal";
```

```suggestion
    pname = "${jdk.pname}-minimal-jre";
```
not sure if we want to keep it or not